samples/cpp/classification_sample_async/main.cpp

```diff
@@ -169,51 +161,23 @@ int main(int argc, char* argv[]) {
                         image_size);
         }
 
-        // -------- Step 8. Do asynchronous inference --------
-        size_t num_iterations = 10;
-        size_t cur_iteration = 0;
-        std::condition_variable condVar;
-        std::mutex mutex;
-
+        // -------- Step 9. Do asynchronous inference --------
         infer_request.set_callback([&](std::exception_ptr ex) {
             if (ex)
                 throw ex;
-            std::lock_guard<std::mutex> l(mutex);
-            cur_iteration++;
-            slog::info << "Completed " << cur_iteration << " async request execution" << slog::endl;
-            if (cur_iteration < num_iterations) {
-                /* here a user can read output containing inference results and put new
-                   input to repeat async request again */
-                infer_request.start_async();
-            } else {
-                /* continue sample execution after last Asynchronous inference request
-                 * execution */
-                condVar.notify_one();
-            }
+            slog::info << "Completed async request execution" << slog::endl;
         });
```

Comment:
why have we remove iterations? In this case user don't see how to call request from its callback

Reply:
restored